docs/user/aws/limits.md

```diff
@@ -53,8 +53,13 @@ the new VPC. If you intend to create more than 20 clusters, you will need to req
 
 ## Security Groups
 
-Each cluster creates 10 distinct security groups. The default limit of 2,500 for new accounts allows for many clusters
-to be created.
+Each cluster creates 4 distinct security groups. The default limit of 2,500 for new accounts allows for many clusters
+to be created. The three security groups created by the default install are:
```

Comment:
nit: drop "three"?  You already count them in the previous line, and the list is short and numbered.  Personally, if we're going to maintain an enumerated list, I'd rather drop "Each cluster creates {number} distinct security groups." as well.